<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="COMPortNumber"/>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Соединение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectButton">
       <property name="text">
        <string>Отключиться</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Стоп</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="speedIndicator">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: &quot;blue&quot;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentIndicator1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: &quot;red&quot;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentIndicator2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: &quot;red&quot;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>581</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid2"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>403</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="referenceSpeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="reverse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>reverse</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="motorState">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: &quot;blue&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor state</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="errorState">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: &quot;blue&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Error state</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="canId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="canIdApply">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="referenceSpeed_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
   </widget>
   <widget class="QPushButton" name="applyAngle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="errorDrive">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>580</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: &quot;red&quot;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="errorNumber">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>610</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: &quot;red&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Drive</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Error</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
